dist/index.js

```diff
@@ -8382,6 +8382,14 @@ async function deploy() {
 
 	const name = slugify(GITHUB_REPOSITORY_NAME);
 
+	console.log('Final Object', {
```

Comment:
do we need to keep these logs in final version?

Reply:
Removed